client/my-sites/domains/domain-management/gsuite/gsuite-purchase-cta/index.jsx

```diff
@@ -112,23 +118,23 @@ class AddGoogleAppsCard extends React.Component {
 							</div>
 						</div>
 
-						<div className="email__add-google-apps-card-logos">
+						<div className="gsuite-purchase-cta__add-google-apps-card-logos">
 							<img alt="G Suite Logo" src="/calypso/images/g-suite/g-suite.svg" />
```

Comment:
@belcherj: it might be a good idea to rename this folder and file as well in your future refactorings.